src/server/server.cc

```diff
@@ -1013,16 +1010,12 @@ void Server::GetClientsInfo(std::string *info) {
 
 void Server::GetMemoryInfo(std::string *info) {
   int64_t rss = Stats::GetMemoryRSS();
-  int memory_lua = lua_gc(lua_, LUA_GCCOUNT, 0) * 1024;
   std::string used_memory_rss_human = util::BytesToHuman(rss);
-  std::string used_memory_lua_human = util::BytesToHuman(memory_lua);
 
   std::ostringstream string_stream;
   string_stream << "# Memory\r\n";
   string_stream << "used_memory_rss:" << rss << "\r\n";
   string_stream << "used_memory_rss_human:" << used_memory_rss_human << "\r\n";
-  string_stream << "used_memory_lua:" << memory_lua << "\r\n";
```

Comment:
We should keep the Lua VM memory usage section. The `used_memory_lua` can be the sum value of the worker's Lua VM memory.

Reply:
Added.